example/googleMapsExample.js

```diff
@@ -190,6 +238,17 @@ function animate() {
 	renderer.render( scene, camera );
 	stats.update();
 
+	// render third person view
+	thirdPersonRenderer.domElement.style.visibility = params.showThirdPerson ? 'visible' : 'hidden';
+	if ( params.showThirdPerson ) {
+
+		cameraHelper.update();
+		cameraHelper.visible = true;
+
+		thirdPersonRenderer.render( scene, thirdPersonCamera );
+
+	}
+
```

Comment:
Do we need to keep all of these debug rendering additions? Ideally this example would just show a simple approach to adding a new google maps globe without too much extra.

Reply:
no need indeed, it was just for me